jp/components/grid_column_pinning.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Grid column pinning
+_description: The Ignite UI for Angular Data Grid control features the fastest, touch-responsive data-rich grid with popular features, including hierarchical and list views.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Data Grid component, Angular Data Grid control, Angular Grid component, Angular Grid control, Angular High Performance Grid, column pinning, pinning, pin
+---
```

Comment:
I think that the Japanese topics are supposed to have a `_language: ja` entry in the metadata?  This would apply to all of the new topics here.

Reply:
Yes, I just copy->pasted the metadata from EN topics